app/src/main/java/edu/hm/cs/ma/todoguru/task/read/ViewTaskFragment.kt

```diff
@@ -49,9 +53,11 @@ class ViewTaskFragment : Fragment() {
         title_task.text = task.title
         description_task.text = task.description
         dueDate_task.text = task.dueDate.toString()
+        reminder_date.text = task.reminder.toString()
```

Comment:
Technical Dept issue: Use direct access to the property of the view or better directly data binding with the view model

Reply:
After discussion in group --> has to be done in next sprint!